CMakeLists.txt

```diff
@@ -59,11 +59,7 @@ option(NE_AVX512_VBMI            "neural_engine: enable AVX512-VBMI"
 option(NE_AVX512_VNNI            "neural_engine: enable AVX512-VNNI"                             OFF)
 option(NE_FMA                    "neural_engine: enable FMA"                                     ON)
 option(NE_AMX                    "neural_engine: enable AMX"                                     OFF)
-
-# in MSVC F16C is implied with AVX2/AVX512
-if (NOT MSVC)
-    option(NE_F16C               "neural_engine: enable F16C"                                    ON)
-endif()
+option(NE_F16C               "neural_engine: enable F16C"                                        ON)
```

Comment:
quite confused about neural_engine here, it should already be deprecated.

Reply:
naming in code will be updated after 0.1